test/preprocess.js

```diff
@@ -100,6 +103,10 @@ describe('preprocess', () => {
   })
 })
 
+const stripStringsLookupTable = (rounds) => {
+  for (const r of Object.values(rounds)) delete r._strings
+}
```

Comment:
Why do we need to do this here? Shouldn't it be unnoticeable from the outside?

Reply:
```
      AssertionError [ERR_ASSERTION]: Expected values to be strictly deep-equal:
+ actual - expected ... Lines skipped

  {
    '0': [
...
        spark_version: undefined,
        start_at: undefined
+     },
+     _strings: Map(3) {
+       '0x000000000000000000000000000000000000dEaD' => '0x000000000000000000000000000000000000dEaD',
+       'UNKNOWN_ERROR' => 'UNKNOWN_ERROR',
+       'ig1' => 'ig1'
      }
    ]
  }
      + expected - actual
```

The refactoring described in https://github.com/filecoin-station/spark-evaluate/pull/86#discussion_r1417504143 would allow us to get the measurements without the strings lookup table.

Reply:
can we define it as non iterable instead?